### DISCUSSION TITLE
Tizen support

### DISCUSSION THREAD

**Discussion by miloush** [oxyplot/oxyplot#2023] [General]
Has anyone managed to get OxyPlot running on Tizen?

**Comment by Jonarw** [comment]
I am guessing that the SkiaSharp or ImageSharp renderers should work fine on Tizen. Of course you'd still need a way of actually presenting the generated bitmap to the user.

**Reply by miloush** [reply]
As in there is no PlotViewRenderer I could use, correct?

**Reply by Jonarw** [reply]
Yes, there is no PlotView implementation provided for Tizen by oxyplot. `SkiaRenderContext` just renders the plot to a `SKCanvas`. You would need some additional means to present that `SKCanvas` to the user. SkiaSharp offers `SKCanvasView` for Tizen, which may be of use here. Plot interaction (zooming, panning etc.) is another topic of course.
